Mathlib/GroupTheory/Perm/Sign.lean

```diff
@@ -13,26 +18,20 @@ import Mathlib.Data.Int.Order.Units
 /-!
 # Sign of a permutation
 
-The main definition of this file is `Equiv.Perm.sign`, associating a `ℤˣ` sign with a
-permutation.
+The main definition of this file is `Equiv.Perm.sign`,
+associating a `ℤˣ` sign with a permutation.
```

Comment:
```suggestion
The main definition of this file is `Equiv.Perm.sign`, associating a `ℤˣ` sign with a
permutation.
```

Reply:
I'm personally a fan of semantic linebreaks (https://sembr.org/).

We don't have a policy in mathlib concerning sembr or justifying text. I would leave this as it is.

Reply:
(That's not the subject here, but I am used to 80 chars terminals, and I find these overflow to 100 chars very disturbing, I get many lines with few symbols, hence less information on my screen.)